### PR TITLE
Update PR#4 

### PR DESCRIPTION
**Pull request by elong0527**
This is to align discussion in PR #4

### CODE REVIEW

File: tlf-disposition.Rmd [hunk 1]
```diff
@@ -107,14 +110,16 @@ n_reason
...
-n_complete <- n_reason %>% filter(row == "Completed")
+n_complete <- n_reason %>% 
+  filter(row == "Completed")
 
 n_complete
 ```
 
 In the code below, we calculate the number and percentage of participants who discontinued the study for different reasons.
-Here we use `paste0("    ", row)` to add some leading space for individual reasons for study discontinuation.
+For display purposes, we use `paste0("    ", row)` to add leading spaces to mimick identation in the final report.
```
Comment (SHAESEN2) [inline tlf-disposition.Rmd:122]: For display purposes,  `paste0(" ", row)` is used to add leading spaces to mimick identation in the final report

File: tlf-disposition.Rmd [hunk 2]
```diff
@@ -1,41 +1,43 @@
 # Disposition {#disposition}
 
 Following [ICH E3 guidance](https://database.ich.org/sites/default/files/E3_Guideline.pdf), 
-we need to summarize the accounting of all participants who entered the study, 
-in Section 10.1, Disposition of Participants.
+we need to summarize the accounting of all participants who entered the study in Section 10.1, Disposition of Participants.
```
Comment (SHAESEN2) [inline tlf-disposition.Rmd:4]: a summary needs to be provided to account for

File: tlf-disposition.Rmd [hunk 3]
```diff
@@ -134,7 +138,7 @@ tbl_disp <- bind_rows(n_rand, n_complete, n_disc, n_reason) %>%
 tbl_disp
 ```
 
-We start to define the format of the output.
+In the below code, formatting of the final table is defined. 
 We highlighted items that is not discussed in the previous discussion.
```
Comment (SHAESEN2) [inline tlf-disposition.Rmd:142]: Items that were not discussed in the previous sections, are highlighted below.

**Review by SHAESEN2 — CHANGES_REQUESTED** [review]
Removal of 'we' + ensure active phrasing

File: tlf-disposition.Rmd (hunk 1 above)
Comment (elong0527) [reply tlf-disposition.Rmd:122]:
```suggestion
For display purposes, `paste0("    ", row)` is used to add leading spaces to mimick identation in the final report.
```

File: tlf-disposition.Rmd (hunk 2 above)
Comment (elong0527) [reply tlf-disposition.Rmd:4]:
```suggestion
A summary needs to be provided to account for of all participants who entered the study in Section 10.1, Disposition of Participants.
```

File: tlf-disposition.Rmd (hunk 3 above)
Comment (elong0527) [reply tlf-disposition.Rmd:142]:
```suggestion
Items that were not discussed in the previous sections, are highlighted below.
```

**Pull request merged**